matrix/inverse_of_matrix.py

```
@@ -0,0 +1,54 @@
+from decimal import Decimal
+
+
+def inverse_of_matrix(matrix: list):
+    """
+    A matrix multiplied with its inverse gives the identity matrix.
+    This function finds the inverse of a 2x2 matrix.
+    If the determinant of a matrix is 0, its inverse does not exist.
+
+    Sources for fixing inaccurate float arithmetic:
+    https://stackoverflow.com/questions/6563058/how-do-i-use-accurate-float-arithmetic-in-python
+    https://docs.python.org/3/library/decimal.html
+
+    >>> inverse_of_matrix([[2, 5], [2, 0]])
+    [[0.0, 0.5], [0.2, -0.2]]
+    >>> inverse_of_matrix([[2.5, 5], [1, 2]])
+    'Does not exist'

```

This should raise a ValueError() exception instead of returning a string.